packages/app-store/around/api/_getAdd.ts

```diff
@@ -19,7 +21,7 @@ export async function getHandler(req: NextApiRequest, res: NextApiResponse) {
     key: {},
   });
 
-  return { url: "/apps/installed" };
+  return { url: getInstalledAppPath({ variant, slug }) };
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.